tests/backend/test_logger.py

```diff
@@ -0,0 +1,65 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+from typing import Mapping
+from hypercorn.config import Config as HyperConfig
+
+import base64
+
+from hypercorn.typing import ResponseSummary, HTTPScope
+
+from parsec.backend.asgi.logger import ParsecLogger
+
+
+def _create_http_scope(author: bytes = bytes(), add_author: bool = True) -> HTTPScope:
+    scope = HTTPScope()
+    scope["type"] = "http"
+    scope["method"] = "GET"
+    scope["query_string"] = b"/"
+    scope["path"] = ""
+    scope["scheme"] = ""
+    scope["headers"] = [(b"Author", author)] if add_author else []
```

Comment:
```suggestion
def _create_http_scope(author: Optional[bytes] = None) -> HTTPScope:
    scope = HTTPScope()
    scope["type"] = "http"
    scope["method"] = "GET"
    scope["query_string"] = b"/"
    scope["path"] = ""
    scope["scheme"] = ""
    scope["headers"] = [(b"Author", author)] if author is not None else []
```
Wouldn't be simpler by using an `Optional` field ?